components/molecules/InsightHeader/insight-header.tsx

```diff
@@ -6,31 +6,45 @@ import getRepoInsights from "lib/utils/get-repo-insights";
 import { useRepositoriesList } from "lib/hooks/useRepositoriesList";
 
 import CardRepoList from "../CardRepoList/card-repo-list";
+import Button from "components/atoms/Button/button";
+import { FaEdit } from "react-icons/fa";
```

Comment:
nit: move third-party imports to the top of the file

Reply:
I assume you mean delete the empty lines between the imports right?

Reply:
Those are fine. Mainly just move things we import from packages to the top, followed by imports external to this folder in general, then local relative imports.

Reply:
like this?